includes/theme-compat/header.php

```diff
@@ -14,43 +14,43 @@
-                                               width="300" height="80"> </a>
+                <a href="/" title="logo"> <img class="site-header-branding" src="/img/logo.png" alt="logo" width="300"
+                                               height="80"></a>
             </div>
             <nav class="site-header-navigation">
                 <ul class="menu">
-                    <li class="menu-item <?php if (isset($page_name) && $page_name == 'home') { ?> current-menu-item <?php } ?>">
+                    <li class="menu-item <?php if ( isset( $page_name ) && $page_name == 'home' ) { ?> current-menu-item <?php } ?>">
                         <a href="/index.php">Home</a>
                     </li>
-                    <?php if (isset($_SESSION['id_user'])) { ?>
-                        <?php if (isset($_SESSION['is_admin']) && $_SESSION['is_admin']) { ?>
-                            <li class="menu-item <?php if (isset($page_name) && $page_name == 'dashboard') { ?> current-menu-item <?php } ?> ">
+					<?php if ( isset( $_SESSION['id_user'] ) ) { ?>
+						<?php if ( isset( $_SESSION['is_admin'] ) && $_SESSION['is_admin'] ) { ?>
+                            <li class="menu-item <?php if ( isset( $page_name ) && $page_name == 'dashboard' ) { ?> current-menu-item <?php } ?> ">
                                 <a href="/dashboard.php">Dashboard</a>
                             </li>
-                        <?php } ?>
-                        <li class="menu-item <?php if (isset($page_name) && $page_name == 'profile') { ?> current-menu-item <?php } ?>">
+						<?php } ?>
+                        <li class="menu-item <?php if ( isset( $page_name ) && $page_name == 'profile' ) { ?> current-menu-item <?php } ?>">
                             <a href="/profile.php">My Profile</a>
                         </li>
                         <li class="menu-item">
                             <a href="/signout.php">Sign Out</a>
                         </li>
-                    <?php } ?>
-                    <?php if (empty($_SESSION['id_user'])) { ?>
-                        <li class="menu-item <?php if (isset($page_name) && $page_name == 'register') { ?> current-menu-item <?php } ?>">
+					<?php }
+					if ( empty( $_SESSION['id_user'] ) ) { ?>
+                        <li class="menu-item <?php if ( isset( $page_name ) && $page_name == 'register' ) { ?> current-menu-item <?php } ?>">
                             <a href="/register.php">Register</a>
                         </li>
-                        <li class="menu-item <?php if (isset($page_name) && $page_name == 'login') { ?> current-menu-item <?php } ?> ">
+                        <li class="menu-item <?php if ( isset( $page_name ) && $page_name == 'login' ) { ?> current-menu-item <?php } ?> ">
                             <a href="/login.php">Login</a>
                         </li>
-                    <?php } ?>
+					<?php } ?>
                 </ul>
```

Comment:
When you have IF conditions wrapping an HTML it's always better to use the alternative syntax for them:

https://www.php.net/manual/en/control-structures.alternative-syntax.php

Reply:
```php
<?php if ( some condition ) : ?>
    <div>Lorem ipsum dolor</div>
<?php endif; ?>
```
Or sometimes you might need to init a variable inside the IF, then you do like this:
```php
<?php
if ( some condition ) :
    $my_variable = 'hello';
    ?>
    <div><?php echo $my_variable; ?></div>
    <?php
endif;
```